backend/WebApi/Controllers/UsersController.cs

```diff
@@ -11,10 +11,9 @@
 
 namespace WebApi.Controllers
 {
-    //TODO: add role access control
     [Route("api/[controller]")]
     [ApiController]
-    [Authorize]
+    //[Authorize]
```

Comment:
Sollte glaub wieder rein oder?

Reply:
Oh shit ja hast du recht, ich hatte ein Problem mit dem JWT und konnte es in Swagger nicht einfügen trotz Bearer+Token

Reply:
ISt der jetzt wieder

Reply:
Nope